index_tutorials.py

```diff
@@ -7,6 +7,11 @@
 
 async def main():
     db = sqlite_utils.Database("content.db")
+    db["tutorials"].create({
+        "path": str,
+        "title": str,
+        "body": str,
+    }, pk="path")
```

Comment:
Had to explicitly create this table here, because otherwise when Datasette tried to render the `/tutorials` page this template fragment would throw an error:

https://github.com/simonw/datasette.io/blob/a705655cf57ca26a57d67e98ec7e9040e809fe58/templates/_more_tutorials.html#L2-L7